File: ITWORK.API/Modules/Organization.cs
using System;
using System.Collections.Generic;

namespace ITWORK.API.Modules
{
    public class Organization
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime DateCreated { get; set; }
        public string PublicID { get; set; }
        public User User { get; set; }
        public int UserId { get; set; }
        public ICollection<OrganizationPhoto> OrganizationPhotos { get; set; }
        public ICollection<OrganizationFollow> OrganizationFollows { get; set; }
    }
}

File: ITWORK-SPA/src/app/_models/organization.ts
import { OrganizationPhoto } from './organizationPhoto';
import { OrganizationHeadPhoto } from './organizationHeadPhoto';

export interface Organization {
    id: number;
    name: string;
    dateCreated: Date;
    description: string;
    userId: number;
    organizationPhotoUrl: string;
    organizationHeadPhotoUrl: string;
    organizationPhotos?: OrganizationPhoto[];
    organizationHeadPhotos?: OrganizationHeadPhoto[];
}
